packages/dcos-integration-test/extra/test_ec2.py

```diff
@@ -145,7 +82,17 @@ def test_move_external_volume_to_new_agent(cluster):
         cluster.deploy_marathon_app(read_app, **deploy_kwargs)
         cluster.destroy_marathon_app(read_app['id'])
     finally:
-        try:
-            _delete_ec2_volume(test_label)
-        except Exception as ex:
-            raise Exception("Failed to clean up volume {}: {}".format(test_label, ex)) from ex
```

Comment:
Let's keep this exception handler so that there's a clear error message with the volume name in it if the test fails to clean up the volume.